src/PostgrestTransformBuilder.ts

```diff
@@ -267,7 +267,7 @@ export default class PostgrestTransformBuilder<
       .filter(Boolean)
       .join('|')
     // An Accept header can carry multiple media types but postgrest-js always sends one
-    const forMediatype = this.headers['Accept']
+    const forMediatype = 'application/json'
```

Comment:
@laurenceisla There's also `csv` and `geojson()` transforms. Would `explain()` work with those too?

Reply:
https://supabase.com/docs/reference/javascript/db-csv

Reply:
Maybe the fix is to assign application/json whenever accept is not defined

Reply:
Yes, a conditional works nicely. A test was missing for these transformations, so I'll add it.

Reply:
nvm... it needs new changes to make explain and csv/geojson to work together in TypeScript. I added the conditional in the meantime.